src/main/java/hvac/database/entities/WeatherSnapshot.java

```diff
@@ -0,0 +1,92 @@
+package hvac.database.entities;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Table( name = "WEATHER_SNAPSHOT",
+    indexes = {
+        @Index(columnList = "ID", name = "weather_id"),
+        @Index(columnList = "WEATHER_TIMESTAMP", name="weather_timestamp")
+    })
+public class WeatherSnapshot {
+    private Long id;
+    private LocalDateTime date;
+    private float temperatureKelvin;
+    private float pressureHPa;
+    private float absoluteHumidity;//in kg/m^3
```

Comment:
Może zamiast pisać jednostki w nazwach, ustalmy że robimy wszystko w SI? Szczególnie, że czasem zmienne mają jednostki w nazwie, a czasem nie. Dla jednolitości (ja też muszę w Coordinatorze zmienić stałe na SI, no może poza milisekundami, bo tutaj one są raczej podstawowym kwantem czasu i lepiej używać ich zamiast ułamków sekund) lepiej trzymać się jednych jednostek, bo inaczej może być jak tu: https://en.wikipedia.org/wiki/Mars_Climate_Orbiter

Reply:
ok, poprawię to, zrobię wszędzie SI